dev/bots/prepare_package.dart

```diff
@@ -2,28 +2,29 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:async';
 import 'dart:convert';
 import 'dart:io';
+import 'dart:typed_data';
 
 import 'package:args/args.dart';
 import 'package:path/path.dart' as path;
+import 'package:http/http.dart' as http;
 
 const String CHROMIUM_REPO =
     'https://chromium.googlesource.com/external/github.com/flutter/flutter';
 const String GITHUB_REPO = 'https://github.com/flutter/flutter.git';
+const String MINGIT_FOR_WINDOWS_URL = 'https://storage.googleapis.com/flutter_infra/mingit/'
+    '603511c649b00bbef0a6122a827ac419b656bc19/mingit.zip';
 
 /// The type of the process runner function.  This allows us to
 /// inject a fake process runner into the ArchiveCreator for tests.
```

Comment:
I believe Todd created package:process to make this easier.

Reply:
Nice.  I'll check it out.  It's [this one](https://github.com/google/process.dart), right?  I found [this other ProcessManager](https://github.com/dart-lang/io), but it was broken when it came to immediate stdout and stderr output.

Reply:
Yes, this one: https://pub.dartlang.org/packages/process